### ISSUE TITLE
[Bug] Sync after disconnect seems to fail

### ISSUE THREAD

**Issue by frankcorneliusmartin** [vantage6/vantage6#654]
**Describe the bug**
2 different logs, 2 different senarios. Task_id should not be in this request.
```python
2023-05-10 12:33:46 - socket         - INFO     - (Re)Connected to the /tasks namespace
2023-05-10 12:33:46 - client         - DEBUG    - Making request: GET | https://petronas.vantage6.ai:443/result | {'state': 'open', 'include': 'task', 'node_id': 43, 'task_id': 1931}
2023-05-10 12:33:46 - socket         - INFO     - Node <demo-kit - demo-D> joined room <collaboration_8>
2023-05-10 12:33:46 - socket         - INFO     - Node <demo-kit - demo-D> joined room <collaboration_8_organization_22>
2023-05-10 12:33:46 - node           - DEBUG    - []
2023-05-10 12:33:46 - node           - INFO     - Received 0 tasks
2023-05-10 12:33:46 - socket         - DEBUG    - Tasks synced again with the server...
```

This one results in a single already running task..
```python
2023-05-10 13:36:29 - socket         - INFO     - (Re)Connected to the /tasks namespace
2023-05-10 13:36:29 - client         - DEBUG    - Making request: GET | https://petronas.vantage6.ai:443/result | {'state': 'open', 'include': 'task', 'node_id': 28, 'task_id': 1962}
2023-05-10 13:36:29 - socket         - INFO     - Node <demo-kit-C - demo-kit Node> joined room <collaboration_8>
2023-05-10 13:36:29 - socket         - INFO     - Node <demo-kit-C - demo-kit Node> joined room <collaboration_8_organization_14>
2023-05-10 13:36:30 - client         - INFO     - Decrypting input
2023-05-10 13:36:30 - node           - DEBUG    - [{'organization': {'id': 14, 'link': '/organization/14', 'methods': ['GET', 'PATCH']}, 'started_at': '2023-05-10T13:32:26.036133+00:00', 'node': {'name': 'demo-kit-C - demo-kit Node', 'ip': '10.76.0.170', 'config': [8863, 8864], 'status': 'online', 'id': 28}, 'finished_at': None, 'ports': [{'port': 49174, 'id': 2117, 'label': None}], 'log': None, 'task': {'name': 'subtask', 'children': None, 'database': '', 'init_user': 42, 'collaboration': {'id': 8, 'link': '/collaboration/8', 'methods': ['DELETE', 'PATCH', 'GET']}, 'status': 'active', 'run_id': 1157, 'parent': {'id': 1934, 'link': '/task/1934', 'methods': ['GET', 'DELETE']}, 'description': 'task from container on node_id=43', 'complete': False, 'initiator': 22, 'id': 1962, 'image': 'harbor.carrier-mu.src.surf-hosted.nl/carrier/health_ri_demo'}, 'result': None, 'assigned_at': '2023-05-10T13:30:44.006136+00:00', 'status': 'active', 'input': b'\x80\x04\x95\x14\x00\x00\x00\x00\x00\x00\x00}\x94\x8c\x06method\x94\x8c\x04init\x94s.', 'id': 2395}]
2023-05-10 13:36:30 - node           - INFO     - Not starting task 1962 - subtask as it is already running
2023-05-10 13:36:30 - node           - INFO     - Received 0 tasks
```

**Comment by frankcorneliusmartin** [comment]
Locally tested, and did not find the same..
```
2023-05-11 07:09:28 - engineio.client - WARNING  - WebSocket connection was closed, aborting
2023-05-11 07:09:28 - socket         - INFO     - Disconnected from the server
2023-05-11 07:09:40 - socket         - INFO     - Node <Small Organization - ZEPPELIN Node> joined room <collaboration_1>
2023-05-11 07:09:40 - socket         - INFO     - Node <Small Organization - ZEPPELIN Node> joined room <collaboration_1_organization_3>
2023-05-11 07:09:40 - socket         - INFO     - (Re)Connected to the /tasks namespace
2023-05-11 07:09:40 - client         - DEBUG    - Making request: GET | http://host.docker.internal:5000/api/result | {'state': 'open', 'include': 'task', 'node_id': 9}
2023-05-11 07:09:41 - node           - DEBUG    - []
2023-05-11 07:09:41 - node           - INFO     - Received 0 tasks
2023-05-11 07:09:41 - socket         - DEBUG    - Tasks synced again with the server...
2023-05-11 07:09:41 - node           - DEBUG    - Sharing node configuration: {'encryption': False, 'allowed_algorithms': ['^harbor2.vantage6.ai/[a-zA-Z]+/[a-zA-Z]+', 's102099/sleeper'], 'allowed_users': [99], 'allowed_orgs': ['IKNL']}
```

**Comment by frankcorneliusmartin** [comment]
Fixed by merging #657
